fcrepo-export/src/test/java/org/fcrepo/exporter/ExporterTest.java

```diff
@@ -146,6 +152,18 @@ public void testExportContainer() throws Exception {
         Assert.assertTrue(exporter.wroteFile(new File("target/rdf/rest/1.jsonld")));
     }
 
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
This seems a bit oblique. Is there a reason that the one test that does throw an exception can't use the `expected` attribute on `@Test`?